x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/api.ts

```diff
@@ -231,7 +231,7 @@ export const duplicateRules = async ({ rules }: DuplicateRulesProps): Promise<Bu
         rule_id: undefined,
         updated_at: undefined,
         updated_by: undefined,
-        enabled: rule.enabled,
+        enabled: false,
```

Comment:
Just a thought. I wish we had some endpoints for duplicating rules so we could write a set of integration tests for that. But currently this is implemented on the client side, so we have to rely on unit and cypress tests for now.